pennylane_lightning/lightning_kokkos/lightning_kokkos.py

```diff
@@ -466,8 +463,12 @@ def expval(self, observable, shot_range=None, bin_size=None):
                 Expectation value of the observable
             """
             if observable.name in [
+                "Identity",
```

Comment:
Do we need identity here? Shouldn't we be able to no-op it?

Reply:
removed